### PR TITLE
Fixes #19629: Update technique to use RUDDER_REPORTS_DB_URL and RUDDER_REPORTS_DB_PASSWORD

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19629

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/rudder-service-relayd/1.0/common/relayd.conf.tpl [hunk 1]
```diff
@@ -48,12 +48,8 @@ retention = "1day"
 
 [output.database]
 {{#classes.root_server}}
-url = "postgres://{{{vars.rudder_postgresql.db_user}}}@{{{vars.rudder_postgresql.host}}}/{{{vars.rudder_postgresql.db_name}}}"
-password = "{{{vars.rudder_postgresql.postgres_passwd}}}"
-{{/classes.root_server}}
-{{^classes.root_server}}
-url = "postgres://user@host/rudder"
```
Comment (amousset) [inline techniques/system/rudder-service-relayd/1.0/common/relayd.conf.tpl:55]: no need for this anymore with >= 6.1 relays

File: techniques/system/common/1.0/promises.st [hunk 2]
```diff
@@ -12,6 +12,20 @@ bundle common rudder_roles
...
 
+&if(INITIAL)&
+      # no kind for initial policies
+      "root_server"   expression => strcmp("root","${rudder_roles.uuid}");
+      "policy_server" expression => "root_server";
+&else&
+      # kind, i.e. "node", "relay" or "root"
+      "rudder_kind_&RUDDER_NODE_KIND&" expression => "any";
+      "root_server"                    expression => "rudder_kind_root";
+      "policy_server"                          or => { "rudder_kind_relay", "rudder_kind_root" }
```
Comment (Fdall) [inline techniques/system/common/1.0/promises.st:23]:
```suggestion
      "policy_server"                          or => { "rudder_kind_relay", "rudder_kind_root" };
```

**Comment by amousset** [comment]
PR updated with a new commit

**Review by Fdall — APPROVED** [review]

**Pull request merged**